src/Urgency.ts

```diff
@@ -41,18 +41,26 @@ export class Urgency {
         }
 
         switch (task.priority) {
-            // High
+            // Highest
             case '1':
+                urgency += 1.5 * Urgency.priorityCoefficient;
+                break;
+            // High
```

Comment:
Ok, I will fix that.
Maybe we could set highest to 9.0, so it will be the same as "due today"?
For lowest it seems that we need to assign a negative value. I'm not sure if it makes any difference if it will be "-1" or something else?

Reply:
> Ok, I will fix that.
> Maybe we could set highest to 9.0, so it will be the same as "due today"?
> For lowest it seems that we need to assign a negative value. I'm not sure if it makes any difference if it will be "-1" or something else?

Thanks. I've thought about this a bit more and I'm gonna suggest to you "don't agonise it over it, just pick some values for the new priorities that are different from the previous ones.

This is because, whilst you are working on this, on `main` I will do some work to auto-generate the text in the docs that shows how the urgency values... I may even generate a chart as well.

Then, once this PR is merged, it will be easy to visualise the values for the new priorities, and decide how to tweak the multipliers for the new priorities to give something that looks plausible and useful.